cdc/processor.go

```diff
@@ -806,6 +810,14 @@ func createSchemaStorage(pdEndpoints []string, credential *security.Credential,
 }
 
 func (p *processor) addTable(ctx context.Context, tableID int64, replicaInfo *model.TableReplicaInfo) {
+	globalResolvedTs := atomic.LoadUint64(&p.sinkEmittedResolvedTs)
+	/*
+		if replicaInfo.StartTs < globalResolvedTs {
+			log.Fatal("cannot add table which of startTs is less than the global resolved, please report a bug",
+				zap.Int64("tableID", tableID), zap.Any("replicaInfo", replicaInfo),
+				zap.Uint64("globalResolvedTs", globalResolvedTs))
+		}
+	*/
```

Comment:
How about removing '/*' and '*/'?

Reply:
There's a bug that's causing this place to fatal. I'm working on it.

Reply:
I will fix it in next PR